### DISCUSSION TITLE
How to remove two pools being shown as one group .

### DISCUSSION THREAD

**Discussion by MMIharish** [cardano-community/pool_groups#22] [Q&A] [answered]
How to remove two pools being shown as one group in cexplorer? Can I this repo file? update cluster and single operator file?
@rdlrt @ashisherc @ilap

**Comment by rdlrt** [comment, marked as answer]
Which group are we talking about?
There is an update of this repo expected soon (awaiting release of koios-1.0.10 for integration as source in a couple of weeks), alongwith switching to adastat as second source for complete list. With the update you might already see pool not showing under cluster status. 

The current status of repo was paused for newer updates until these revamps were ready
